### ISSUE TITLE
Hiden config files are not found by local_vimrc

### ISSUE THREAD

**Issue by mcepl** [sgur/vim-editorconfig#18]
When I have this ``.editorconfig`` file:
```ini
[*.rst]
local_vimrc = .config.vim
```
Values from the file ``.config.vim`` are not loaded. When I rename the file (and fix the value in ``.editorconfig``, of course) to ``config.vim``, everything works. Exactly the example from [README](https://github.com/sgur/vim-editorconfig/blob/master/README.md) (using ``.local.vimrc``) doesn’t work for me. Using vim8-1290 on RHEL-7.

**Comment by sgur** [comment]
I found that `local_vimrc` file is not execute when the current directory isn't the directory which `.editorconfig` is located.

I fixed this bug in 3050a5a3567aa048674bc68a24d1f828a395f0d9. So please check with the HEAD version.

**Comment by mcepl** [comment]
```
matej@mitmanek: petunia_dursley (master *+%)$ cat .editorconfig 
[*.rst]
# spell_enabled = true
# spell_language = cs
local_vimrc = .config.vim
matej@mitmanek: petunia_dursley (master *+%)$ cat .config.vim 
setlocal spell spelllang=cs
matej@mitmanek: petunia_dursley (master *+%)$ vim -c 'echo &spelllang ' 17_zpatky_londyn.rst 
matej@mitmanek: petunia_dursley (master *+%)$
```
The last command still shows ``en``.
